### PR TITLE
Feature/contact us

### PR DESCRIPTION
**Pull request by Kyle-WSU**

### CODE REVIEW

File: src/main/resources/static/webpageFun.js [hunk 1]
```diff
@@ -44,8 +44,75 @@ function loginCheck() {
       });
 
       if(isLoggedOut != null){
-          alert("You have been logged out.");
+        //alert("You have been logged out.");
```
Comment (jacobboesch) [inline src/main/resources/static/webpageFun.js]: Remove commented out code

File: src/main/resources/static/webpageFun.js [hunk 2]
```diff
@@ -44,8 +44,75 @@ function loginCheck() {
...
+  let notVerified = params.get("notVerified");
+  let notAdmin = params.get("notAdmin");
+
+  if(notVerified == null){
+      // change this to be more user friendly upon error
+      document.getElementById("feedbackDisplay").style.display = "block";
+  }
+
+  if(notAdmin == null){
+      //alert("Only Administrators can access this page.");
```
Comment (jacobboesch) [inline src/main/resources/static/webpageFun.js]: Remove commented out code

File: src/main/resources/static/webpageFun.js [hunk 3]
```diff
@@ -44,8 +44,75 @@ function loginCheck() {
...
+  setCookie("openModal", "true", 1);
+  window.location.href= "index.html";
+}
+
+function feedbackLabel() {
+  document.getElementById("feedbackLabel").style.display = "block";
+  setTimeout(function() {
+    document.getElementById("feedbackLabel").style.display = "none";
+    //uncomment when rolled out
+    //window.location.href= "index.html";
```
Comment (jacobboesch) [inline src/main/resources/static/webpageFun.js]: remove commented out code

File: src/main/resources/templates/accessDenied.html [hunk 4]
```diff
@@ -17,14 +86,18 @@
 
 if(notVerified != null){
     // change this to be more user friendly upon error
-    alert("Only verified users can access this page. Please check your email.");
+    //alert("Only verified users can access this page. Please check your email.");
```
Comment (jacobboesch) [inline src/main/resources/templates/accessDenied.html:89]: remove commented out code

File: src/main/resources/templates/accessDenied.html [hunk 5]
```diff
@@ -17,14 +86,18 @@
...
     // change this to be more user friendly upon error
-    alert("Only verified users can access this page. Please check your email.");
+    //alert("Only verified users can access this page. Please check your email.");
+    document.getElementById("adminUser").style.display = "none";
+    document.getElementById("verifedUser").style.display = "block";
 }
 
 if(notAdmin != null){
-    alert("Only Administrators can access this page.");
+    //alert("Only Administrators can access this page.");
```
Comment (jacobboesch) [inline src/main/resources/templates/accessDenied.html:95]: Remove commented out code

File: src/main/resources/templates/viewFeedback.html [hunk 6]
```diff
@@ -3,8 +3,200 @@
...
+              <div class="centered">
+                <div class="feedbackContainer" id="feedbackContainer">
+                  <div class="ui placeholder segment" id="fogCover" style="background: rgba(0,0,0,.7); max-width: 753px; max-height: 349px; overflow-y: auto;">
+                    <h2>Feedback</h2>
+                    <template id="commentTemplate">
+                        <div class="comment">
+                            <div class="text">
+                                  Hello World!
+                            </div>
+                            <!--<button class="ui red inverted button" id="temp" onclick="deleteButton(this.id)">Resolve</button>-->
```
Comment (jacobboesch) [inline src/main/resources/templates/viewFeedback.html]: remove this comment if it's not being used

File: src/main/resources/templates/viewFeedback.html [hunk 7]
```diff
@@ -3,8 +3,200 @@
...
+          // SET INPUT ATTRIBUTE 'type' AND 'value'.
+          button.setAttribute('type', 'button');
+          button.setAttribute('class', 'ui red inverted button');
+          button.setAttribute('value', 'Resolve');
+          button.setAttribute('id', data[i].id);
+
+          // ADD THE BUTTON's 'onclick' EVENT.
+          button.setAttribute('onclick', 'deleteButton(this.id)');
+
+          //$("#temp").attr("id", data[i].id);
```
Comment (jacobboesch) [inline src/main/resources/templates/viewFeedback.html]: remove commented out code

File: src/main/resources/templates/viewFeedback.html [hunk 8]
```diff
@@ -3,8 +3,200 @@
...
+
+        textComponent.innerHTML = format;
+        //document.getElementById("but").onclick = deleteButton(data[i].id);
+/*
+        $(document).ready(function(){
+          $('#but').click(deleteButton(data[i].id));
+        });
+*/
```
Comment (jacobboesch) [inline src/main/resources/templates/viewFeedback.html]: remove this if it's not being used

File: src/main/resources/templates/viewFeedback.html [hunk 9]
```diff
@@ -3,8 +3,200 @@
...
+  }
+
+  $(document).ready(function() {
+    fetchMessages();
+/*
+    $('.ui red inverted button').click(function() {
+     var t = $(this).attr('id');
+     deleteButton(t);
+   });
+*/
```
Comment (jacobboesch) [inline src/main/resources/templates/viewFeedback.html]: Remove if not being used

**Review by jacobboesch — CHANGES_REQUESTED** [review]
For the most part everything looks good. 
However, you need to remove the commented code. 
Also, It looks like the Blood-Donatin.jpg file no longer exists on your branch. When I ran the program the image wasn't there.

**Review by jacobboesch — CHANGES_REQUESTED** [review]
The image folder still doesn't show up. Could you please add the Blood-Donation image back.

**Review by jacobboesch — APPROVED** [review]
Everything looks good 👍 . 
May want to change the email to a real admin email address when we eventually get one.

**Review by Lloyd-Brombach-WSU — APPROVED** [review]

**Review by TheKPham — APPROVED** [review]

**Review by tia-gijo — COMMENTED** [review]
good

**Pull request merged**